{
  "name": "@pidchashyi/try-catch",
  "version": "3.1.3",
  "description": "A TypeScript utility package for elegant error handling with Result types",
  "keywords": [
    "typescript",
    "error-handling",
    "result",
    "try-catch",
    "retry",
    "async",
    "utility"
  ],
  "homepage": "https://github.com/Marian1309/try-catch#readme",
  "bugs": {
    "url": "https://github.com/Marian1309/try-catch/issues"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/Marian1309/try-catch.git"
  },
  "license": "MIT",
  "author": "Marian Pidchashyi",
  "main": "dist/index.js",
  "types": "dist/index.d.ts",
  "files": [
    "src",
    "dist",
    "LICENSE",
    "README.md"
  ],
  "scripts": {
    "build": "tsc",
    "prepare": "npm run build",
    "postpublish": "rimraf -rf dist"
  },
  "devDependencies": {
    "@types/node": "^24.0.3",
    "typescript": "^5.0.0"
  },
  "dependencies": {
    "async_hooks": "^1.0.0"
  }
}
